sql/update.go

```diff
@@ -90,20 +93,68 @@ func (p *planner) Update(n *parser.Update) (planNode, error) {
 		vals = append(vals, val)
 	}
 
+	if len(cols) != len(vals) {
+		panic("columns not equal updated values")
```

Comment:
I've removed this panic. It's really useless given that both slices are created within the same function and equal len(n.Exprs).